File: exs/mundo_1/python/006.py
"""
Desafio 006

Problema: Crie um algoritmo que leia um número e mostre o seu drobro,
          triplo e raiz quadrada.

Resolução do problema:
"""
num = int(input('Informe um valor: '))

dobro = num * 2
triplo = num * 3
raiz = num ** (1 / 2)  # Também pode-se utilizar a função pow(), de sintaxe: pow(num, (1/2))

print('Dobro: {}\nTriplo: {}\nRaiz Quadrada: {:.2f}'.format(dobro, triplo, raiz))

File: exs/mundo_1/ruby/006.rb
=begin
Desafio 006

Problema: Crie um algoritmo que leia um número e mostre o seu drobro,
          triplo e raiz quadrada.

Resolução do problema:
=end
print"Digite um número: "
n = gets.chomp.to_i
dobro = n*2
triplo = n*3
raiz = n**0.5
puts"o dobro de #{n} é #{dobro}."
puts"o triplo de #{n} é #{triplo}."
puts"a raiz quadrada de #{n} é #{raiz}."